### PR TITLE
Add hip17 beacon only test group

### PR DESCRIPTION
**Pull request by vihu**
- Adds 4 new tests for beacon only rewarding with some basic assumptions.
- Update chain vars for testing with new percentages

TODO:

- [x] Check if we have incorrect witness reward normalization, if so hide behind reward_version >= 6
- [x] Remove debugging noise

### CODE REVIEW

File: src/transactions/v1/blockchain_txn_rewards_v1.erl [hunk 1]
```diff
@@ -854,7 +854,10 @@ normalize_witness_rewards(WitnessRewards, #{epoch_reward := EpochReward,
                                             poc_witnesses_percent := PocWitnessesPercent}=Vars) ->
     TotalWitnesses = lists:sum(maps:values(WitnessRewards)),
     ShareOfDCRemainder = share_of_dc_rewards(poc_witnesses_percent, Vars),
-    WitnessesReward = (EpochReward * PocWitnessesPercent) + ShareOfDCRemainder,
+
```
Comment (evanmcc) [inline src/transactions/v1/blockchain_txn_rewards_v1.erl:857]: the changes to this file look like debugging chaff, maybe revert them?

Comment (vihu) [reply src/transactions/v1/blockchain_txn_rewards_v1.erl]: Yup done

**Review by evanmcc — APPROVED** [review]
approved, but please revert the

**Review by evanmcc — APPROVED** [review]

**Comment by vihu** [comment]
> approved, but please revert

Yes, I updated this branch to only have the test file specific changes

**Pull request merged**